test/git/Git.test.ts

```diff
@@ -1,13 +1,13 @@
-import { afterEach, beforeEach, describe, expect, it, jest } from '@jest/globals';
-import * as simpleGitMock from 'simple-git';
+import simpleGitMock from 'simple-git';
```

Comment:
to get mocks workings I imported the default. Defaults are the main difference in terms of mocking.